examples/router/select-input.js

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
```

Comment:
The example is not runnable. See the other examples.

Reply:
The other examples in the `examples` folder are not enough since they don't require external packages. The best I could do, atm, was to make the example runnable out-the-box by using it to replace the `ui.js` file in a newly generated `npx creat-ink-app` project, after installing both dependencies `npm i react-router ink-select-input`. This seems a bit complicated to me. 

My suggestion is remove the example `router/select-input` and integrate it into `docs/recipes/routing-with-react-router.md`, so that gives a better chance at explaing the right runtime and usage